app/javascript/packs/main_vue.js

```diff
@@ -206,6 +206,35 @@ document.addEventListener('turbolinks:load', () => {
       }
     });
   });
+
+  $('#increase-blog-font-size-btn').on('click', function() {
+    var originalFontSize = $('#blog-content-body').css('font-size');
```

Comment:
Precisamos de um filtro mais seletivo pq está afetando os posts relacionados. E em alguns casos não altera o texto da matéria. Algo tipo: $('#blog-content-body .wordpress-theme p,h1,h2,h3,h4,h5 not:.post-cards-section')
Só tem que ver a sintaxe de seleção ali do jquery...